tailwind.config.mjs

```diff
@@ -81,8 +82,18 @@ export default {
         },
       },
       zIndex: {
+        1: '1',
         2: '2',
       },
     },
   },
+  plugins: [
+    plugin(({ addVariant }) => {
+      // Add a custom variant such that the `annotator-collapsed:` modifier
+      // is available. The `Sidebar` logic adds the `.annotator-collapsed`
+      // class to the sidebar frame when it's collapsed. This modifier allows
+      // sub-components to select for that state.
+      addVariant('annotator-collapsed', '.annotator-collapsed &');
```

Comment:
This variant prefix is used to make the bucket bar background mostly transparent when the sidebar is collapsed.

Reply:
We could give this a more obvious name like `sidebar-collapsed`, now that we don't have to worry about collisions with the host page styles.

An alternative approach to all of this would be to have the `Sidebar` class in the annotator modify a property on the `BucketBar` when the sidebar is opened/closed and have the bucket bar controller re-render the component with different styles. This would be less efficient though.

Reply:
> I prefer sidebar-collapsed but the CSS class name assigned by code in Sidebar is annotator-collapsed

Sorry, I was unclear. My thought was that the class name used by the `Sidebar` would change at the same time.

Reply:
Ah! I see. Yes, we can do that but I'd like to do it as follow-up, as the `.annotator-collapsed` selector is relied upon in a few other spots in the legacy SASS.